api/src/legacy_soap_api/legacy_soap_api_client.py

```diff
@@ -264,4 +269,46 @@ def get_simpler_soap_response(self, proxy_response: SOAPResponse) -> SOAPRespons
             f"{simpler_response_xml}\n"
             f"{boundary}"
         ).encode("utf-8")
-        return get_soap_response(data=mime_message, headers=update_headers)
+
+        legacy_tracking_number = self.get_soap_request_dict()["GrantsGovTrackingNumber"]
+        if legacy_tracking_number.startswith("GRANT"):
+            legacy_tracking_number = legacy_tracking_number.split("GRANT")[1]
+        application = self.db_session.execute(
+            select(ApplicationSubmission).where(
+                ApplicationSubmission.legacy_tracking_number == int(legacy_tracking_number)
+            )
+        ).scalar()
+
+        def _gen_response_data(
+            mime_message: bytes, boundary: str, response: requests.Response
+        ) -> Iterator:
+            yield mime_message
+            for chunk in response.iter_content(chunk_size=4000):
+                yield chunk
+            yield b"\n" + boundary.encode("utf-8") + b"--"
+
+        if application:
+            response = requests.get(application.download_path, timeout=10)
+            if response.status_code == 200:
+                return get_soap_response(
+                    data=_gen_response_data(mime_message, boundary, response),
+                    headers=update_headers,
+                )
```

Comment:
Don't build a custom approach to fetching files from s3. The download_path is meant for public users and uses a presigned URL and is a very roundabout way of getting the file.

We have several file utils that can handle this - `file_util.open_stream` is probably what you'd want?